.github/workflows/performance.yml

```diff
@@ -162,6 +162,14 @@ jobs:
         run: |
           npm run env:cli -- rewrite structure '/%year%/%monthnum%/%postname%/' --path=/var/www/${{ env.LOCAL_DIR }}
 
+      - name: Install additional languages
+        run: |
+          npm run env:cli -- language core install de_DE --path=/var/www/${{ env.LOCAL_DIR }}
+          npm run env:cli -- language plugin install de_DE --all --path=/var/www/${{ env.LOCAL_DIR }}
+          npm run env:cli -- language plugin update de_DE --all --path=/var/www/${{ env.LOCAL_DIR }}
+          npm run env:cli -- language theme install de_DE --all --path=/var/www/${{ env.LOCAL_DIR }}
+          npm run env:cli -- language theme update de_DE --all --path=/var/www/${{ env.LOCAL_DIR }}
```

Comment:
Why do the plugin and theme ones need to run `update` immediately after `install`? I'm not familiar and it looks a bit weird (also because it's not the case for core), so it would be good to add a comment to explain.